apps/rule-manager/conf/evolutions/default/12.sql

```diff
@@ -0,0 +1,16 @@
+-- !Ups
+
+ALTER TABLE rules_draft
+    ADD COLUMN rule_order INT UNIQUE;
+
+UPDATE rules_draft
+    SET rule_order = id
+    WHERE true;
```

Comment:
If you don't supply a `WHERE` clause, the update will apply to all rows, so we should be able to remove this line.

```suggestion
    SET rule_order = id;
```

Reply:
I added the `WHERE` clause because when trying to run the query without it I got the message: `Unsafe query: 'Update' statement without 'where' updates all table rows at once` and needed to explicitly click another `Execute` button for it to run. But imagine this is just IntelliJ trying to help me out, and in production it would run fine?

Reply:
That's right, this is just IntelliJ making sure you're not accidentally ruining your own day 😀

Reply:
Yes, this sounds like an IntelliJ warning when you try to manually run the query against the DB (in IntelliJ's DB console). I assume when Play runs the evolution it runs the query without this prompt (both locally and in production) - but may be wrong!